client/e2e/poms/profile.ts

```diff
@@ -48,14 +48,10 @@ export class ProfilePOM {
     await this.buttonSubmit.click();
     // Wait for the success message to appear
     await this.page.waitForSelector('div:has-text("✅ Success")', {
-      timeout: 11000,
+      timeout: 15000,
     });
```

Comment:
For your consideration, I haven't tested it much in CI but in my `industry_user_admin` pr I had some issues with waiting for elements after a network request or clicking a button/link that sends you to another page. I have had some success with using `await this.page.waitForLoadState("networkidle");`.

I'll approve this but let me know if this makes any difference for you, I saw you have another PR updating some tests.

I use it for the click functions at the bottom of this file:

https://github.com/bcgov/cas-registration/pull/1071/files

Reply:
@marcellmueller FYI, from `Gemini`

I'd advise against using await this.page.waitForLoadState("networkidle"); directly in your updateSuccess function for the following reasons:

Unnecessary Wait: Waiting for network idle might introduce unnecessary overhead if your success message doesn't necessarily depend on external resources loading after the form submission.
Flaky Tests: networkidle can be flaky in certain scenarios, especially with applications that have ongoing background network activity. It might lead to false test failures even if the success message appears as expected.

Reply:
> @marcellmueller FYI, from `Gemini`
> 
> I'd advise against using await this.page.waitForLoadState("networkidle"); directly in your updateSuccess function for the following reasons:
> 
> Unnecessary Wait: Waiting for network idle might introduce unnecessary overhead if your success message doesn't necessarily depend on external resources loading after the form submission. Flaky Tests: networkidle can be flaky in certain scenarios, especially with applications that have ongoing background network activity. It might lead to false test failures even if the success message appears as expected.

Yeah I did some reading on it and looks like it's supposed to be used when changing pages and not during async requests on the page. I suppose that's why I had success with it. Do you think I should avoid it altogether?